wordcamp.dev/provision/vvv-init.sh

```diff
@@ -62,6 +62,25 @@ else
 
 	git -C $SITE_DIR/wp-content/plugins/camptix pull origin master
 
+	wme_noroot wp plugin install supportflow            --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install polldaddy              --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install liveblog               --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install camptix-trustpay       --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install camptix-trustcard      --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install camptix-mercadopago    --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install camptix-kdcpay-gateway --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install wp-super-cache         --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install custom-content-width   --path=$SITE_DIR/wordpress
+
+	wme_noroot wp plugin activate wordpress-importer         --network --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin activate wordcamp-docs              --network --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin activate wordcamp-dashboard-widgets --network --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin activate camptix-network-tools      --network --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin activate wp-super-cache             --network --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin activate custom-content-width       --network --path=$SITE_DIR/wordpress
+
+	wme_noroot wp plugin deactivate wordcamp-api --network --path=$SITE_DIR/wordpress
+
```

Comment:
I'm not excited about changing the state of various plugins within the provisioning script. Especially because if you manually change the state of a plugin at some later point, and then after that re-run the provisioner, it will change the plugin state back again without your knowledge.

The best solution, I think, would be to import the database dump, modify only the plugin states, and then export it again and add the dump file to the diff here.

Another option might be to just disable the notification about plugins not being in the right state, since they are really only intended for the Production site anyway.